src/rez/serialise.py

```diff
@@ -73,13 +73,18 @@ def open_file_for_write(filepath, mode=None):
             with atomic_write(filepath, overwrite=True, **encoding) as f:
                 f.write(content)
 
-        except WindowsError as e:
-            if attempt == 0:
+        except OSError as e:
+            # If we are writing to the network, we can't ensure atomicity
+            if e.errno == 22:
```

Comment:
Error no. 22 is an invalid filepath, right? What does that have to do with network paths?

Reply:
The goal is to handle the sync method error of atomicwrites: 

>  File "/usr/local/lib/python3.7/site-packages/rez/vendor/atomicwrites/__init__.py", line 44, in _sync_directory
>   _proper_fsync(fd)
>OSError: [Errno 22] Invalid argument

This is ugly, but atomicwrites doesn't handle properly this error itself